lighthouse-extension/app/src/popup.js

```diff
@@ -37,6 +37,32 @@ document.addEventListener('DOMContentLoaded', _ => {
   const optionsList = document.body.querySelector('.options__list');
   const okButton = document.getElementById('ok');
 
+  function getLightHouseVersion() {
```

Comment:
nit: we aren't always consistent with this, but lighthouse is one word, so should be `getLighthouseVersion()`